targets_test.go

```diff
@@ -31,14 +34,22 @@ func TestStructTarget_Ensure(t *testing.T) {
 			test: func(v reflect.Value, err error) {
 				assert.NoError(t, err)
 				require.False(t, v.IsNil())
-				s := v.Interface().([]string)
+
+				s, ok := v.Interface().([]string)
+				if !ok {
+					panic(fmt.Sprintf("interface %v should be castable into []string", s))
```

Comment:
The single assign form for casting will panic with a nice error message when the cast fails, no need to do it manually.

Reply:
then, we probably have to add a nolint :/. There s not config as far as I can see.

Reply:
Let's disable `forcetypeassert`. I'm not sure about why this would be a good idea to enforce it, maybe for programs that should never-ever panic? Not our case here.

Reply:
Don't you think it enforces contributor to consciously panic? I feel it's a safeguard that is a little noisy, but can catch some errors at some point?

Reply:
I sometimes refer to the [Uber golang style guide](https://github.com/uber-go/guide/blob/master/style.md#dont-panic). Definitely not THE reference, but usually a good point of view. This is a test though, I know it doesnt really matter.